### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by lfelipe1501**
Update Readme for fix image demo

### CODE REVIEW

File: themes/atomic/README.md [hunk 1]
```diff
@@ -159,11 +159,18 @@ Three environment variables can be defined to rearrange the segments order. The
 
 `___ATOMIC_BOTTOM="char"`
 
-## Follow me
+### Contact / Social Media
 
-I'm on the social media.
+*Get the latest News about Web Development, Open Source, Tooling, Server & Security*
```
Comment (nwinkler) [inline themes/atomic/README.md]: Sorry, but I don't feel comfortable with this. This open source project is not a platform for promoting and advertising your own social media profiles.

File: themes/atomic/README.md [hunk 2]
```diff
@@ -159,11 +159,18 @@ Three environment variables can be defined to rearrange the segments order. The
...
-* [Luis Felipe](https://www.facebook.com/lfelipe1501) on Facebook.
+[![Twitter](https://github.frapsoft.com/social/twitter.png)](https://twitter.com/lfelipe1501)
+[![Facebook](https://github.frapsoft.com/social/facebook.png)](https://www.facebook.com/lfelipe1501)
+[![Github](https://github.frapsoft.com/social/github.png)](https://github.com/lfelipe1501)
+
+### Development by
+
+Developer / Author: [Luis Felipe Sánchez](https://github.com/lfelipe1501)
+
+Company: [lfsystems](https://www.lfsystems.com.co)
```
Comment (nwinkler) [inline themes/atomic/README.md]: ...and neither is it a platform for advertising your own business.

Comment (lfelipe1501) [reply themes/atomic/README.md]: ok, I'm going to remove these lines

**Comment by nwinkler** [comment]
Great - thanks for understanding!

**Pull request merged**